TestingDangerApp/ViewController.swift

```diff
@@ -1,26 +1,33 @@
-//
-//  ViewController.swift
-//  TestingDangerApp
-//
-//  Created by PicPay Eduardo on 7/23/19.
-//  Copyright © 2019 PicPay Eduardo. All rights reserved.
-//
-
 import UIKit
 
 class ViewController: UIViewController {
     lazy var label: UILabel = {
         let label = UILabel()
-        label.text = "bla bla bla"
+        label.text = "Olá mundo!"
+        label.font = UIFont.systemFont(ofSize: 24)
+        label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .gray
-        // Do any additional setup after loading the view.
+        view.backgroundColor = .yellow
+        addComponents()
+        layoutComponents()
+    }
+    
+    func addComponents() {
+        view.addSubview(label)
+    }
+    
+    func layoutComponents() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/4eb8a63644473b39dfe145c57954c4eb567b3f70/TestingDangerApp/ViewController.swift#L23"></span>An ‘other_method’ should not be placed amongst the type content(s) ‘view_life_cycle_method’.<br />
<code>type_contents_order</code> <code>ViewController.swift:23</code></td>
    </tr>
  </tbody>
</table>